source/source/nebula/ecs/system/DrawSpriteSystem.hx

```diff
@@ -28,17 +28,13 @@ class DrawSpriteSystem extends AbstractSystem
             {
                 if(sprite.isRepeating)
                 {
-                    var s:FlxBackdrop = new FlxBackdrop(sprite.image);
-                    sprite.sprite =  s;
-                    this.state.add(s);
+                    sprite.sprite = new FlxBackdrop(sprite.image) ;
                 }
                 else
                 {
-                    var s:FlxSprite = new FlxSprite();
-                    s.loadGraphic(sprite.image);
```

Comment:
Ah, I didn't notice that. The [api docs](http://api.haxeflixel.com/flixel/FlxSprite.html) suggest:

> `SimpleGraphic` The graphic you want to display (OPTIONAL - for simple stuff only, do NOT use for animated images!).

This will be a problem later, but I guess we can cross that bridge later.